R/rdeephaven/README.md

```diff
@@ -0,0 +1,72 @@
+# The Deephaven R Client
+
+The Deephaven R client is an R package that enables R users to interface with a Deephaven server and perform various
+server-side operations from the comfort of RStudio or any other R interface.
+
+## What can the R client do?
+
+The Deephaven Client currently provides three basic functionalities:
+
+1. Connect to a Deephaven server
+   -   with anonymous authentication (no username or password)
+   -   with basic authentication (username and password)
+   -   with custom authentication (general key-value credentials)
+
+2. Interface with Deephaven Tables on the server
+   -   Retrieve references to tables on the server
+   -   Pull table data into an [Arrow RecordBatchReader](https://arrow.apache.org/docs/r/reference/RecordBatchReader.html),
+an [Arrow Table](https://arrow.apache.org/docs/r/reference/Table.html),
+a [dplyr Tibble](https://tibble.tidyverse.org),
+or an [R Data Frame](https://stat.ethz.ch/R-manual/R-devel/library/base/html/data.frame.html)
+   -   Create new tables on the server from an [Arrow RecordBatchReader](https://arrow.apache.org/docs/r/reference/RecordBatchReader.html),
+an [Arrow Table](https://arrow.apache.org/docs/r/reference/Table.html),
+a [dplyr Tibble](https://tibble.tidyverse.org),
+or an [R Data Frame](https://stat.ethz.ch/R-manual/R-devel/library/base/html/data.frame.html)
+   -   Bind server-side tables to variable names, enabling access from outside the current R session
+
+3. Run scripts on the server
+   -   If the server is equipped with a console, run a script in that console
+   -   Currently, Python and Groovy are supported
+
+## Installation
+
+Currently, the R client is only supported on Ubuntu 20.04 or 22.04 and must be built from source.
+
+1. Choose a directory where the client source code will live. Here, the source code will be downloaded into a new directory called `deephaven`.
+   Navigate into that directory and clone this subdirectory of `deephaven-core` using git's sparse-checkout:
+   ```bash
+   mkdir deephaven
+   cd deephaven
+   git init
+   git remote add -f origin https://github.com/deephaven/deephaven-core.git
+   git config core.sparseCheckout true
+   echo "R/rdeephaven" >> .git/info/sparse-checkout
+   git pull origin main
+   ```
+2. Now, navigate into the source code `lib` directory and build the C++ client and dependencies:
+   ```bash
+   cd R/rdeephaven/lib
+   chmod +x build-cpp.sh
+   ./build-cpp.sh
+   ```
+3. With the C++ client installed, start an R console and build and install the R client as follows:
+   ```bash
+   R
+   >> install.packages("/path/to/rdeephaven", repos=NULL, type="source")
+   ```
```

Comment:
```suggestion
   ```bash
   R
   ```
   ```r
   install.packages("/path/to/rdeephaven", repos=NULL, type="source")
   ```
```